ml-notebooks/image_classification/pcam/02_2_vertex_ai_pcam.ipynb

```diff
@@ -0,0 +1,979 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "6e2c084da6b2"
+   },
+   "source": [
+    "# Using Vertex AI to train an image classification model\n",
+    "\n",
+    "<div class=\"alert alert-block alert-info\">\n",
+    "Run the <a href=\"xxx\"><code>00_pcam_setup.ipynb notebook</code></a> first, before running this one.  You'll need the settings info from that notebook.\n",
+    "</div>\n",
+    "\n",
+    "## Introduction\n",
+    "\n",
+    "This notebook shows some examples of how to use [Vertex AI](https://cloud.google.com/vertex-ai/docs) for training a machine learning model. \n",
+    "\n",
+    "Notebook `02_1_vertex_ai_pcam` showed how to define and submit a **model training job**; then how to upload and deploy that model for serving; and then how to send prediction requests to the deployed model's *Endpoint*. It also showed how to create and use a Managed Tensorboard instance during training, and how to log information about the training run to the Vertex Experiments API.\n",
+    "\n",
+    "This notebook shows how to set up a [**hyperparameter tuning**](https://en.wikipedia.org/wiki/Hyperparameter_optimization) job using that same model; and how to set up and run a **distributed multi-node training** job.\n",
+    "\n",
+    "Then, a following set of notebooks show how to use Vertex Pipelines to define an ML workflow for data preprocessing, training, model evaluation, and deployment.\n",
+    "\n",
+    "The example code is here: https://github.com/verily-src/terra-solutions-ml.\n",
+    "\n",
+    "### Estimated cost of running this notebook\n",
+    "\n",
+    "The dataset used for the examples in this notebook is fairly large, as is the base model architecture, and training using the notebook's default configurations will take a number of hours. Time estimates are added in each section.\n",
+    "\n",
+    "The model training works best with GPU(s)— it runs fine using only CPUs, but training will take an even longer time. For this example, the notebook itself doesn't need GPUs; instead they'll be used by Vertex AI.\n",
+    "\n",
+    "The HP tuning example should cost ~37USD to run, and the distributed training example < 7.5USD, not including the cost of the notebook instance.\n",
```

Comment:
Same ask here, can you specify what service the ~37 is charged to?

Reply:
Same-- Vertex AI services in the 'native' project. I'll clarify for all the notebooks. 

I won't attempt a further breakdown, which they can get from their billing info.
Essentially the majority of the charges are training costs (& in particular here, the HP job takes a while and does many training runs during its search).  For some of the other notebooks there's a charge for deploying the trained model too (only briefly deployed if they clean up afterwards), and for running Pipelines. There are also tiny GCS charges.